pkg/sloop/webfiles/sloop_ui.js

```diff
@@ -406,7 +374,8 @@ function createResourceBar(d) {
         .attr("x", isLabelRight ? sx - 5 : sx + w + 5)
         .attr("fill", "black")
         .style("text-anchor", isLabelRight ? "end" : "start")
-        .style("dominant-baseline", "hanging");
+        .style("dominant-baseline", "hanging")
+        .style("font-size", "14");
```

Comment:
Can we use CSS for the font sizes?  If not, can we define the value at the top of this file so its easy to find and tweak?